packages/react-charting/src/components/StackedBarChart/MultiStackedBarChart.base.tsx

```diff
@@ -282,14 +292,30 @@ export class MultiStackedBarChartBase extends React.Component<IMultiStackedBarCh
       if (data.chartData!.length === 0) {
         bars.push(
           <g key={0} className={this._classNames.noData} onClick={this._redirectToUrl.bind(this, href)}>
-            <rect key={0} x={'0%'} y={0} width={'100%'} height={barHeight} fill={palette.neutralLight} />
+            <rect
+              key={0}
+              id={this._barIdEmptyPartToWhole}
```

Comment:
These are placeholder bars, not empty bars.

Reply:
done